src/test/cli/rbd/help.t

```diff
@@ -625,13 +625,14 @@
     -o [ --options ] arg            device specific options
   
   rbd help device list
-  usage: rbd device list [--device-type <device-type>] [--format <format>] 
-                         [--pretty-format] 
+  usage: rbd device list [--device-type <device-type>] [--options <options>] 
+                         [--format <format>] [--pretty-format] 
```

Comment:
I think I would prefer if we just add `--all` option instead of introducing device specific `--options`. For the `list` command it looks awkward.

Reply:
Sure, I'm fine with it.